script.reddit.reader/resources/language/English/strings.po

```diff
@@ -42,7 +42,7 @@ msgctxt "#32009"
 msgid "Cannot retrieve URL"
```

Comment:
it would be nice if you, at some point, change the folder layout for translations since you're targeting krypton. lang attributes in addon.xml should be `en_GB` (new default). Folders should be resources.language.en_gb instead of just "English". Consider to add it in a new update.